src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -156,6 +156,7 @@ def __init__(self, *args, **kwargs):
 
         self.agent_sessions = None
         self.setup_start_servers = True
+        self.setup_start_agents = True
```

Comment:
Both. Creating that attribute allows me to use the `TestWithServers` class on my test and toggle whether to launch agents.

Reply:
@tanabarr So yes, this is a general _apricot_ improvement to increase flexibility with different test requirements.  A specific example of this would be that the test wants to control when the agents are started instead of having it handled by the framework for it.